services/alertmanager/alertmanager.go

```diff
@@ -130,6 +131,40 @@ receivers:
 		err = ioutil.WriteFile(alertmanagerBaseConfigPath, []byte(defaultBase), 0644) //nolint:gosec
 		svc.l.Infof("%s created: %v.", alertmanagerBaseConfigPath, err)
 	}
+
+	var content []byte
+	_, err = os.Stat(alertmanagerBaseConfigPath)
+	svc.l.Debugf("%s status: %v", alertmanagerBaseConfigPath, err)
```

Comment:
That will be a strange log in case file exists, `/srv/alertmanager/alertmanager.base.yml status: nil`

Reply:
removed, I was also using the wrong file path, changed that